From 9dbc1975c45ec6b941d9b52251be0b356a6add9c Mon Sep 17 00:00:00 2001
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
Date: Mon, 12 Jan 2026 16:34:39 +0200
Subject: [PATCH 112/137] arm64: dts: rockchip: Add frl-enable-gpios to
 rk3576-evb1-v10

In preparation to support HDMI 2.1 FRL operating mode, make use of the
GPIO2_B0 line (labeled HDMI_TX_ON_H in the schematics) to control the
voltage bias for HDMI data lines.

The GPIO line is asserted when operating in FRL mode and deasserted for
TMDS.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 arch/arm64/boot/dts/rockchip/rk3576-evb1-v10.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3576-evb1-v10.dts b/arch/arm64/boot/dts/rockchip/rk3576-evb1-v10.dts
index c5584c26d..9dd5b8152 100644
--- a/arch/arm64/boot/dts/rockchip/rk3576-evb1-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3576-evb1-v10.dts
@@ -300,6 +300,7 @@ &gpu {
 };
 
 &hdmi {
+	frl-enable-gpios = <&gpio2 RK_PB0 GPIO_ACTIVE_LOW>;
 	status = "okay";
 };
 
-- 
2.47.3

